tests/test_wave_01.py

```diff
@@ -123,7 +123,7 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
```

Comment:
What can we assert about the pieces of data a movie holds to confirm it is the movie we expect it to be?